### PR TITLE
Create 0078-use-aws-appconfig-feature-flags.md

### PR DESCRIPTION
**Pull request by pbtruss**

### CODE REVIEW

File: docs/adrs/0078-use-aws-appconfig-feature-flags.md [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+* '+' No additional setup for Platform Team
+* '-' We will at some point ship the wrong code
+* '-' Will break API versioning required by Prime
+
+
+### Option #1: LaunchDarkly
+* '+' Industry standard
+* '+' Feature Rich
+* '-' Expensive. No room in ODC budget
+* '-' Might not be allowed (It is FedRAMP moderate, not IL4)
```
Comment (felipe-lee) [inline docs/adrs/0078-use-aws-appconfig-feature-flags.md:69]: **question**: who could get us an answer on whether it is ok to use or not?

File: docs/adrs/0078-use-aws-appconfig-feature-flags.md [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+LaunchDarkly is the industry standard, but there is no room for it in the budget. Unleash has similar features to AWS AppConfig, but requires more setup and maintenance effort when self-hosted. This leaves AWS AppConfig as the reasonable choice.
+
+
+## Pros and Cons of the Alternatives
+
+### Option #0: Do Nothing
+* '+' No change to App Eng workflow
+* '+' No additional setup for Platform Team
+* '-' We will at some point ship the wrong code
+* '-' Will break API versioning required by Prime
```
[the remark this answers is not in the document]
Comment (pbtruss) [reply docs/adrs/0078-use-aws-appconfig-feature-flags.md:62]: The other feedback I have gotten on this ADR has made it clear that there are Application Engineering concerns I have not taken into account. The current plan is that the next steps for this ADR will be handled within the App Eng practice. I agree that feature flags and API versioning interact in potentially exciting ways and we should think through that carefully.

File: docs/adrs/0078-use-aws-appconfig-feature-flags.md [hunk 3]
```diff
@@ -0,0 +1,85 @@
...
+* Option #0: Do Nothing. Stay the course, don't make any changes, figure out what to do when problems arise
+* Option #1: LaunchDarkly: This is the industry standard feature flag platform.
+* Option #2: Unleash: Open source, can be self-hosted
+* Option #3: AWS AppConfig: we're already using lots of AWS, what's a little more?
+
+## Decision Outcome
+
+Chosen Alternative: Option #3: AWS AppConfig
+
+LaunchDarkly is the industry standard, but there is no room for it in the budget. Unleash has similar features to AWS AppConfig, but requires more setup and maintenance effort when self-hosted. This leaves AWS AppConfig as the reasonable choice.
```
[the remark this answers is not in the document]
Comment (pbtruss) [reply docs/adrs/0078-use-aws-appconfig-feature-flags.md:53]: I have heard that LaunchDarkly for government sites is about $50k/yr, and it seems like the AWS feature is unlikely to cost that much under normal use. As you point out, the cost approval process is also a factor.

File: docs/adrs/0078-use-aws-appconfig-feature-flags.md (hunk 1 above)
Comment (pbtruss) [reply docs/adrs/0078-use-aws-appconfig-feature-flags.md:69]: If I was going to chase that down, I would start with the project ISSM. If they can't make the call, they know who can.

**Comment by pbtruss** [comment]
This has been superseded by https://github.com/transcom/mymove-docs/pull/365 and should not be merged.

**Pull request closed without merging**